TwoPaneLayout/README.md

```diff
@@ -103,21 +103,24 @@ The pane mode affects when two panes are shown for TwoPaneLayout. By default, wh
 enum class TwoPaneMode {
     TwoPane,
     HorizontalSingle,
-    VerticalSingle
+    VerticalSingle,
+    SinglePane
 }
 ```
 
-There are three pane modes available for TwoPaneLayout:
+There are four pane modes available for TwoPaneLayout:
 
-- `TwoPane` - default mode, always shows two panes regardless of the orientation
-- `HorizontalSingle` - shows one big pane when in the horizontal orientation (combines top/bottom panes)
+- `TwoPane` - default mode, always shows two panes, regardless of the orientation
+- `HorizontalSingle` - shows one big pane when the fold is horizontal (combines top/bottom panes)
 
     <img src="screenshots/single-horizontal.png" width=500 alt="HorizontalSingle pane mode on a dual-screen device">
 
-- `VerticalSingle` - shows one big pane when in the vertical orientation (combines left/right panes)
+- `VerticalSingle` - shows one big pane when the fold is vertical (combines left/right panes)
 
     <img src="screenshots/single-vertical.png" width=500 alt="VerticalSingle pane mode on a foldable device">
 
+- `SinglePane` - always shows one pane, regardless of the orientation
+
```

Comment:
Great idea, I updated the readme and header comments!

Reply:
I realized I didn't address the "tell when we show two panes" part - do you think this table explains that clearly?

> This table shows each pane mode and when two panes will be shown:

| Pane mode | Small window without separating fold | Portrait large window / horizontal separating fold | Landscape large window / vertical separating fold |
|---|---|---|---|
| `TwoPane` |  | ✅ | ✅ |
| `HorizontalSingle` |  |  | ✅ |
| `VerticalSingle` |  | ✅ |  |
| `SinglePane` |  |  |  |

Reply:
Love it! This is even clearer than the description 🔝
Makes sense to add the `otherwise` option? 
e.g. This table shows each pane mode and when two panes will be shown: 
to
This table shows each pane mode and when two panes will be shown. When empty, one pane will be shown:

Or something like that?

Reply:
Ooh I love the emoji idea, I'll change to that instead!